main/ZgatewayBT.ino

```diff
@@ -948,16 +955,18 @@ void launchDiscovery() {
 
 void PublishDeviceData(JsonObject& BLEdata) {
   if (abs((int)BLEdata["rssi"] | 0) < minRssi) { // process only the devices close enough
-    JsonObject& BLEdataOut = process_bledata(BLEdata);
```

Comment:
You are proposing that we don't need to copy the return of process data and BLEdata will be enriched directly if I'm correct?

Reply:
We never copy it. It's simply misleading to have and use a return value. This is not a class to return back with *this.

Reply:
ok thanks